packages/pds/src/context.ts

```diff
@@ -235,6 +239,14 @@ export class AppContext {
     const entrywayAgent = cfg.entryway
       ? new AtpAgent({ service: cfg.entryway.url })
       : undefined
+    let entrywayAdminAgent: AtpAgent | undefined
+    if (cfg.entryway && secrets.entrywayAdminToken) {
+      entrywayAdminAgent = new AtpAgent({ service: cfg.entryway.url })
+      entrywayAdminAgent.api.setHeader(
+        'authorization',
+        basicAuthHeader('admin', secrets.entrywayAdminToken),
+      )
+    }
```

Comment:
FYI this can now be written using an `Agent` instead of an `AtpAgent` (since we don't need the session management stuff)

```suggestion
    const entrywayAdminAgent =
      cfg.entryway && secrets.entrywayAdminToken
        ? new Agent({
            service: cfg.entryway.url,
            headers: {
              authorization: basicAuthHeader('admin', secrets.entrywayAdminToken),
            },
          })
        : undefined
```

Reply:
doesn't look like the agent constructor takes headers so this ended up looking basically exactly the same except `Agent` instead of `AtpAgent`. I think i'll leave as is for now until we switch all the other agents on the the context over to the `Agent` class